docs/sources/flow/getting-started/opentelemetry-to-lgtm-stack.md

```diff
@@ -119,7 +104,7 @@ loki.write "default" {
 }
 ```
 
-To use Loki with basic-auth, which is required with GrafanaCloud Loki, you must configure the [loki.write][] component. You can get the Loki config from the Loki **Details** page in the [GrafanaCloud Portal][]:
+To use Loki with basic-auth, which is required with GrafanaCloud Loki, you must configure the [loki.write][] component. You can get the Loki configuration from the Loki **Details** page in the [GrafanaCloud Portal][]:
```

Comment:
sanity check if GrafanaCloud one word? I saw it a few places and it seemed odd

Reply:
I missed that one :-( It's supposed to be Grafana Cloud (with a space). I fixed it throughout the topic (was repeated several times)